Description: Add categories to the static info files generated by the texi files.
Forwarded: not yet
Author: Thomas Weber <tweber@debian.org>
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -25,6 +25,7 @@
 @include macros.texi
 
 @ifinfo
+@dircategory Math
 @format
 START-INFO-DIR-ENTRY
 * Octave: (octave).           Interactive language for numerical computations.
--- a/doc/liboctave/liboctave.texi
+++ b/doc/liboctave/liboctave.texi
@@ -21,6 +21,7 @@
 @setfilename liboctave.info
 
 @ifinfo
+@dircategory Math
 @format
 START-INFO-DIR-ENTRY
 * liboctave: (liboctave). Octave C++ Classes
